tests/test_basic.py

```diff
@@ -0,0 +1,73 @@
+"""Unit tests"""
+
+import glob
+import os
+import tempfile
+import unittest
```

Comment:
do we care about consistency in test frameworks among library and library studies? just trying to get in front of this now vs later if we're talking about adding testing to these.

Reply:
I took the same approach here I did with the WIP data-metrics study. I just copy-and-pasted the test loading framework from that to here. So there probably is some value in not having that copied all around.

Maybe the Library could provide a little test harness utility method or something similar?

But that can always be follow-on stuff. I've filed [an issue](https://github.com/smart-on-fhir/cumulus-library/issues/219) about it.